lib/gollum/views/helpers.rb

```diff
@@ -2,31 +2,63 @@
 
 module Precious
   module Views
+    module RouteHelpers
+      ROUTES = {
+        'gollum' => {
+          :last_commit_info => 'last-commit-info',
```

Comment:
Stylistically, it's becoming more common to write `last_commit_info: 'last-commit-info'` (instead of the `=>` notation).